tests/test_plugin.py

```diff
@@ -214,5 +214,55 @@ def test_it_adapts_splits_based_on_new_and_deleted_tests(
         ]
 
 
+class TestHasExpectedOutput:
+    def test_prints_splitting_summary(self, example_suite, capsys):
+        result = example_suite.inline_run("--splits", "1", "--group", 1)
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Running group 1/1 (10/10) tests" in outerr.out
+
+    def test_prints_splitting_summary_when_group_missing(self, example_suite, capsys):
+        result = example_suite.inline_run("--splits", "1")
```

Comment:
IMO, ideally, this would not run the tests at all as we it's quite clearly a mistake from the user 🤔 Same with `--group` when `--splits` is missing

Reply:
Yes, I thought so too but assumed this was a deliberate choice (since the case was handled specifically).
I'll check if it's possible to exit with an error code in this case instead.

Reply:
This was fixed in my latest commit